lib/helpers/reports/forChannel.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const convertStandups = require('./convertStandups');
+const generateFields  = require('./generateFields');
+
+function defaultMessage(channelName) {
+  return 'Todays standup for <#'+channelName+'>';
+}
+
+module.exports = function(channel, standups) {
+  let channelName = channel.name;
+  let attachments = convertStandups(standups);
+
+  attachments.unshift({
```

Comment:
I think not unshifting would be better and easier to understand. I think it would be good to get to a pretty dumb representation of this.

I am not sure I would do this work right now, because I can feel a much better abstraction for building these interactions in the future. I don't think we are there yet. Probably after you do your work and refactors.

I would leave behind some refactor-encouraging mess until other areas of the application are refactored. Then we can really see what we have and what would work best.